pkg/cli/login/error_translation.go

```diff
@@ -22,6 +23,8 @@ You can also run this command again providing the path to a config file directly
 `
 	tlsOversizedRecordMsg = `Unable to connect to %[2]s using TLS: %[1]s.
 Ensure the specified server supports HTTPS.`
+	invalidServerURLMsg = `Seems you passed an HTML page (console?) instead of Kubernetes API URL.
```

Comment:
Let's keep it consistent, since we're using `server URL` upon login let's call it like so in here. So proposed message:
`Seems you passed an HTML page (console?) instead of server URL, verify provided address and try again.`

Reply:
updated, 
```
error: response content-type: found text/html
Seems you passed an HTML page (console?) instead of server URL.
Verify provided address and try again.
```